### PR TITLE
Bc 315 navbar

### PR DESCRIPTION
**Pull request by mendo94**
As a user, I want to be able to navigate to the following pages:

landing/home page (bc icon)

project portal

how bc works

about us

As well as being able to:

open the chat dialog 

open the account dropdown

From anywhere on the site so that I can easily access these features and pages. 

AC’s
Update nav links to reflect those mentioned above & still in figma 

Design nav to be like the figma 

Screen Shot 2023-08-13 at 10 22 49 PM

### CODE REVIEW

File: src/layout/Nav/Nav.tsx [hunk 1]
```diff
@@ -122,44 +116,45 @@ const AuthorizedNavLinks = ({
...
-        {(visibleChat || !visibleChat) && (
-          <ChatIconBadge
-            isChatBadgeUpdated={isChatBadgeUpdated}
-            setIsChatBadgeUpdated={setIsChatBadgeUpdated}
+      <div className='message-container'>
+        <div className='messages-icon' ref={chatRef}>
+          <BsFillChatLeftTextFill
+            size={23}
+            className='chat-icon'
+            onClick={() => toggleChatBox()}
```
Comment (yeezick) [inline src/layout/Nav/Nav.tsx:124]: suggestion to turn this into a [named function vs anon callback ](https://www.geeksforgeeks.org/difference-between-anonymous-and-named-functions-in-javascript/#)

File: src/layout/Nav/Nav.scss [hunk 2]
```diff
@@ -99,38 +108,35 @@ nav {
...
+    padding: 6px 12px;
+    text-decoration: none;
   }
 
   .log-in {
-    background-color: #fefefe;
-    border-radius: 10px;
-    color: #000;
-    padding: 15px 30px;
+    background: var(--white-ffffff, #fefefe);
```
Comment (yeezick) [inline src/layout/Nav/Nav.scss:131]: wondering about these vars, why do we have to declare the hex code and a var name?

File: src/App.tsx [hunk 3]
```diff
@@ -15,13 +15,18 @@ import { AllProjects, CreateProject, ProjectDetails } from 'screens/Project'
 import { AvailabilityDemoScreen } from 'screens/TempFeatures/Availability'
 import { CalendarScreen } from 'screens/Calendar/Calendar'
 import './App.css'
+import AboutUs from 'layout/Nav/Links/AboutUs'
+import HowTo from 'layout/Nav/Links/HowTo'
+import ProjectPortal from 'layout/Nav/Links/ProjectPortal'
```
Comment (yeezick) [inline src/App.tsx:20]: We should create the AboutUs and HowTo screens in the screens/landing directory

Project portal is unique and not an actual page but more of a portal to the project features when a user is logged in

For example, there is no sidebar in the landing, about, or how to screens but when the user clicks "project portal" they are routed to the project details screen with the sidebar always open.

Comment (yeezick) [reply src/App.tsx:20]: and they should be exported as consts, not default exports

**Review by yeezick — CHANGES_REQUESTED** [review]
only one worth waiting for is the comment about screens and project portal

File: src/App.tsx (hunk 3 above)
Comment (mendo94) [reply src/App.tsx:20]: Okay agree about the project portal, the screens I am not following the example you provided. Send help

File: src/layout/Nav/Nav.tsx (hunk 1 above)
Comment (mendo94) [reply src/layout/Nav/Nav.tsx]: moved to a named function

File: src/layout/Nav/Nav.scss (hunk 2 above)
Comment (mendo94) [reply src/layout/Nav/Nav.scss]: you never know if you need a backup to your backup color, jk I copied the style straight from Figma. Removed vars.

File: src/App.tsx (hunk 3 above)
Comment (mendo94) [reply src/App.tsx]: moved to the Landing directory, moved project portal to redux project slice, exported as consts

**Review by yeezick — APPROVED** [review]

**Pull request merged**